exporter/awsxrayexporter/translator/aws.go

```diff
@@ -61,6 +71,8 @@ func makeAws(attributes map[string]string, resource pdata.Resource) (map[string]
 			switch key {
 			case semconventions.AttributeCloudProvider:
 				cloud = value.StringVal()
+			case "cloud.infrastructure_service":
```

Comment:
As these are destined to be replaced by constants anyways, I'd go ahead and have private constants like `AttributeCloudInfrastructureService` defined in this file

Reply:
That's fair, will replace